test/framework/vsphere.go

```diff
@@ -713,34 +645,54 @@ func WithBottlerocketFromRelease(release *releasev1.EksARelease, kubeVersion any
 
 func (v *VSphere) WithBottleRocketForRelease(release *releasev1.EksARelease, kubeVersion anywherev1.KubernetesVersion) api.ClusterConfigFiller {
 	return api.VSphereToConfigFiller(
-		api.WithTemplateForAllMachines(v.templateForRelease(anywherev1.Bottlerocket, release, kubeVersion)),
+		api.WithTemplateForAllMachines(v.templateForDevRelease(anywherev1.Bottlerocket, kubeVersion)),
 	)
 }
 
 func optionToSetTemplateForRelease(osFamily anywherev1.OSFamily, release *releasev1.EksARelease, kubeVersion anywherev1.KubernetesVersion) VSphereOpt {
 	return func(v *VSphere) {
 		v.fillers = append(v.fillers,
-			api.WithTemplateForAllMachines(v.templateForRelease(osFamily, release, kubeVersion)),
+			api.WithTemplateForAllMachines(v.templateForDevRelease(osFamily, kubeVersion)),
```

Comment:
same here, we changing this? this is supposed to take any release as an argument and now it's forcing to only use the dev release

Reply:
My bad! I thought both the release would be same only. Changed it back.